platform/admin/ui/src/main/webapp/templates/module/ModuleDetail.layout.handlebars

```diff
@@ -14,7 +14,7 @@
 <div class="page-layout full-height well">
     <div class="header clearfix">
         <div class="btn btn-success pull-right">
-            Export
+            Export System Configurations
```

Comment:
@lessarderic and @garrettfreibott, I prefer descriptive.  In this case it sounds like `Export Configuration` is actually the most accurate.

People could think that System means "only system"